src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from 'react';
+import { wait } from '../../utils/fetchClient';
+import { Error } from '../../types/Error';
+
+type Props = {
+  errors: Error[];
+  setErrors: (cb: (prev: Error[]) => Error[]) => void;
+};
+
+export const Notification: React.FC<Props> = ({ errors, setErrors }) => {
+  const [isHidden, setIsHidden] = useState(false);
+  const { title, isImportant } = errors[0];
+
+  const closeError = () => {
+    setIsHidden(true);
+    wait(900).then(() => {
+      setErrors(prev => prev.slice(1));
+      setIsHidden(false);
+    });
+  };
+
+  useEffect(() => {
+    const intervalId = setInterval(closeError, 3000);
```

Comment:
The useEffect's dependency array should include closeError, so that the timer is reset when a new error is added.